Xamarin.Forms.Platform.Android/Renderers/FormsEditText.cs

```diff
@@ -41,4 +41,20 @@ public override bool RequestFocus(FocusSearchDirection direction, Rect previousl
 
 		internal event EventHandler OnKeyboardBackPressed;
 	}
+
+	[Obsolete("As of 2.4 this class has been replaced with Xamarin.Forms.Platform.Android.FormsEditText")]
```

Comment:
Can we say `"EntryEditText is obsolete as of version 2.4.0. Please use Xamarin.Forms.Platform.Android.FormsEditText instead."` ? Just for consistency with the other obsolete messages. :)